table partition: tidb_enable_exchange_partition variable description (#11342)
pingcap/docs-cn: system-variables.md

- 当集群开启只读模式后，所有用户（包括 `SUPER` 用户）都无法执行可能写入数据的 SQL 语句，除非该用户被显式地授予了 `RESTRICTED_REPLICA_WRITER_ADMIN` 权限。
- 拥有 `RESTRICTED_VARIABLES_ADMIN` 或 `SUPER` 权限的用户可以修改该变量。如果用户开启了[安全增强模式 (Security Enhanced Mode)](/system-variables.md#tidb_enable_enhanced_security)，则只有 `RESTRICTED_VARIABLES_ADMIN` 权限的用户才能修改该变量。

> ### `tidb_enable_exchange_partition` <span class="version-mark">从 v6.3.0 版本开始引入</span>

Author: ```suggestion
### `tidb_enable_exchange_partition`
```

- 作用域：SESSION | GLOBAL
- 是否持久化到集群：是
- 类型：布尔型
- 默认值：`ON`

> - 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#partition-management) 特性。默认值为 `ON`，即默认开启该功能。

Reviewer: ```suggestion
- 该变量用于设置是否启用 [`exchange partitions with tables`](/partitioned-table.md#分区管理) 特性。默认值为 `ON`，即默认开启该功能。
```

> - 当 `exchange partitions with tables` 被禁用时，如果你尝试使用 `exchange partitions with tables`，TiDB 会返回告警。

Reviewer 2: 警告？

Author: OK fixed

> 该变量的值始终为 'ON'，设置成 'OFF' 也不会生效。该变量将在今后的版本移除。

Author: ```suggestion
该变量自 v6.3.0 开始废弃，其取值将固定为默认值 `ON`，即默认开启 `exchange partitions with tables`。
```

> 该变量自 v6.3.0 开始废弃，其取值将固定为默认值 `ON`，即默认开启 `exchange partitions with tables`。

Reviewer: ```suggestion
- 该变量自 v6.3.0 开始废弃，其取值将固定为默认值 `ON`，即默认开启 `exchange partitions with tables`。
```

> - 当该变量设置为 `OFF`，即该功能禁用时，如果你尝试使用 `exchange partitions with tables`，TiDB 会返回警告。

Reviewer 2: 这句话删了吧，目前保持一直打开，麻烦英文的也同步删除下。

Author: 如果用户设置成了 OFF 会发生什么？

> - 当该变量设置为 `OFF`，即该功能禁用时，如果你尝试使用 `exchange partitions with tables`，TiDB 会返回警告。

Author: ```suggestion
```

> - 当该变量设置为 `OFF`，即该功能禁用时，如果你尝试使用 `exchange partitions with tables`，TiDB 会返回警告。

Author: ```suggestion
- 当该变量设置为 `OFF` 时不会生效。
```
